config/unmarshal.go

```diff
@@ -33,99 +33,9 @@ func newViper(allowEnvironmentOverrides bool) *viper.Viper {
 		v.AutomaticEnv()
 	}
 
-	// Set zeroed defaults for all the config settings so that Viper knows what environment variables
```

Comment:
Thrilled to see this go... but it worries me slightly ;P

What happens if I try to define `MM_PLUGINSETTINGS_PLUGINS_*` or some such? Do we correctly not panic / corrupt the config?

Reply:
It doesn't seem to work before or after my changes.